edgraph/access_ee.go

```diff
@@ -155,10 +155,10 @@ func (s *Server) authenticateLogin(ctx context.Context, request *api.LoginReques
 
 	if user == nil {
 		return nil, errors.Errorf("unable to authenticate through password: "+
-			"user not found for id %v", request.Userid)
+			"invalid username or passowrd")
 	}
 	if !user.PasswordMatch {
-		return nil, errors.Errorf("password mismatch for user: %v", request.Userid)
+		return nil, errors.Errorf("invalid username or password")
```

Comment:
You can use the `x.ErrorInvalidLogin` here 
https://github.com/dgraph-io/dgraph/blob/462a505afe3af8e84452b21b1fb251fd574d81b6/edgraph/access_ee.go#L80